openedx/core/djangoapps/user_api/views.py

```diff
@@ -669,26 +669,58 @@ def _add_terms_of_service_field(self, form_desc, required=True):
         terms_text = _(u"Terms of Service")
         terms_link = u"<a href=\"{url}\">{terms_text}</a>".format(
             url=marketing_link("TOS"),
-            terms_text=terms_text
+            terms_text=terms_text,
         )
 
-        # Translators: "Terms of service" is a legal document users must agree to
-        # in order to register a new account.
-        label = _(
-            u"I agree to the {platform_name} {terms_of_service}."
-        ).format(
-            platform_name=settings.PLATFORM_NAME,
-            terms_of_service=terms_link
-        )
+        # Add privacy policy to terms of service checkbox label and error
+        if settings.REGISTRATION_ADD_PRIVACY_TO_TOS_CHECKBOX:
```

Comment:
I don't think we need to add this conditional for this. All of our instance use the same two link set up for the ToS checkbox or replace the whole login page like CME will do